### DISCUSSION TITLE
How do you properly integrate non article pages?

### DISCUSSION THREAD

**Discussion by Myrmod** [importantimport/urara#22] [Q&A] [answered]
Hello, 

I'd like to add a `privacy` page. How is it supposed to be done? 

I added the page to `FooterConfig` in `general.ts` and a `privacy.svelte.md` to the urara directory. I don't need this article stuff, so what layout am I supposed to use and how do I use it?

Should I create my own layout called `page` or something?

best regards,
Myrmod

**Comment by kwaa** [comment, marked as answer]
You should try adding `flags: ['unlisted']` to FrontMatter.

```md
---
title: Title
flags:
- unlisted
---
```

this way the articles will not be listed on the front page and in the feed, the layout is another matter.
